gourmet/gtk_extras/TextBufferMarkup.py

```diff
@@ -78,26 +80,35 @@ def __init__ (self):
         GObject.GObject.__init__(self)
 
     def set_text (self, txt):
+        if isinstance(txt, bytes):
+            # data loaded from the database are bytes, not str
+            txt = txt.decode()
         Gtk.TextBuffer.set_text(self,"")
         try:
-            self.parsed,self.txt,self.separator = Pango.parse_markup(txt,'\x00')
-        except:
-            print('Problem encountered escaping text: "%s"'%txt)
+            self.parsed, self.attrList, self.txt, self.separator = Pango.parse_markup(txt, -1, '\x00')
+        except Exception as e:  # unescaped text (g-markup-error-quark), eg. contains &amp
```

Comment:
I don't know how to get the right exception Python-type (`g-markup-error-quark`)